roles/fio/README.md

```diff
@@ -0,0 +1,21 @@
+# fio
+
+This role creates a fio container that can be used to generate concurrent i/o's for both block and file storage.
+
+## Privilege escalation
+
+Yes, provoleged cess is required to run this container
+
+## Parameters
+
+* `dev-type`        : fs or blk - Specify what type of device to run I/Os on 
+* `block-device`    : Block device path to run I/O on (eg: /dev/sda or /dev/sdb)
+
+Optional Paramters: 
+* `time-to-run` : Specify if you want to run time based tests (in seconds)
+* `file-size`   : Incase of File system, the file size to be written, Default is 1G if not specified.
+* `numjobs`     : Optional, Number of Parallel fio jobs, Default is 1
```

Comment:
Let's keep these parameters in-line with the framework i.e
```
neerali_fio_dev_type
neerali_fio_file_size
```
when passing to the script they can follow what is expected by the script.

Reply:
Fixed